roles/core/pxe_stack/tasks/client_os/Suse.yml

```diff
@@ -19,7 +19,7 @@
     mode: 0644
     owner: "{{ pxe_stack_apache_user }}"
     group: "{{ pxe_stack_apache_group }}"
-    validate: /usr/bin/xmllint --noout --relaxng /usr/share/YaST2/schema/autoyast/rng/profile.rng %s
+    # validate: /usr/bin/xmllint --noout --relaxng /usr/share/YaST2/schema/autoyast/rng/profile.rng %s
```

Comment:
Why has this validation check been commented out?

Reply:
Good question 🙂 
Issue is, these files do not exist on other distributions than opensuse. So I need to comment them for now. However, I kept it here commented, in hope I can find a solution in the future.

Reply:
No worries - that makes sense. If xmllint exists as a package on other distributions then perhaps it could be a package that is installed by the role? Of course, we could also have to provide a copy of `/usr/share/YaST2/schema/autoyast/rng/profile.rng` from SLES/OpenSUSE.

Reply:
That would make sense, you are right.
I will open an issue for that, to do it in a second step.

Reply:
https://github.com/bluebanquise/bluebanquise/issues/688